Lombiq.BaseTheme.Samples/Assets/Styles/pages/blog-post.scss

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------------
+// Blog Post page rules
+// ================================
+// This stylesheet is only injected as a resource when the server is displaying
+// a BlogPost content type (see: Services/ResourceFilters.cs).
+// -----------------------------------------------------------------------------
+
+@import "../abstracts/mixins";
+
+// Here we are styling built-in fields and parts.
+.blog-post {
+    & > p:first-child {
+        @include margin-top-gutter;
```

Comment:
Here and in others files VS tells "Undeclared mixin".

Reply:
VS? More like a Very Sad attempt at tooling amirite? (the mixins are imported at line 8)

Reply:
:D Let's continue this under the https://github.com/Lombiq/Orchard-Base-Theme/pull/15#discussion_r852446152 thread.